### DISCUSSION TITLE
Why are some YT-Videos blocked, others not?

### DISCUSSION THREAD

**Discussion by dermaxem** [AdguardTeam/AdGuardHome#6955] [Q&A] [answered]
Hello there,
iam not sure if AdGuardHome as somthing to do with it. this YT-Video seems not reachable by desktop-pc, but on mobiel viua VPN.
https://www.youtube.com/watch?v=Z5GsCrfBA5E
if i try to open i see:
This video is restricted. Please check your Google Workspace administrator restrictions and/or your network administrator restrictions.
-
#Other YT-Videos on my desktop working fine.
If i open this on my mobile (not connectet to adguard) i can see the video.

**Comment by Dynamic5912** [comment, marked as answer]
Because you're using a VPN on the mobile device. This is bypassing local DNS

**Reply by dermaxem** [reply]
Thank thee for this quick answer. If i check the URL "youtube.com" i can see that it is not filtered. So could it be, that YT notice in some way, that i filter den Adds, and this is the reason for the message i see?
